frictionless/checks/regulation.py

```diff
@@ -59,6 +59,80 @@ def validate_row(self, row):
     }
 
 
+class table_dimensions(Check):
+    """Check for minimum and maximum table dimensions
+
+    API      | Usage
+    -------- | --------
+    Public   | `from frictionless import checks`
+    Implicit | `validate(checks=[{"code": "table-dimensions", min_rows, max_rows, min_fields, max_fields}])`
+
+    Parameters:
+       descriptor (dict): check's descriptor
+
+    """
+
+    code = "table-dimensions"
+    Errors = [errors.TableDimensionsError]
+
+    def __init__(
+        self, descriptor=None, *, min_rows=-1, max_rows=-1, min_fields=-1, max_fields=-1
```

Comment:
- defaults need to be None as in other checks (try `check.to_dict` with these defaults)
- please add `number_rows/fields` as it's asked here - https://github.com/frictionlessdata/frictionless-py/issues/820#issuecomment-825526182 (it's basically a shorthand for setting min/max to the same number)

Reply:
Changed to None